docs/spec_draft.md

```diff
@@ -127,3 +127,79 @@ specification.
 %z = stablehlo.add %x, %y : tensor<2x2xf32>
 // %z: [[6, 8], [10, 12]]
 ```
+
+## stablehlo.maximum
+
+`stablehlo.maximum(lhs, rhs) -> result`
+
+### Semantics
+
+Performs element-wise max operation on tensors `lhs` and `rhs` and produces a
+`result` tensor. For Floating-point element type, implements IEEE 754 semantics:
+Returns the larger of two arguments, propagating `NaN`s and treating `-0` as
+less than `+0`.
+
+### Arguments
```

Comment:
I've just noticed that the "Structure of an Op’s Specification" section says "Operands". I'd say that "Operands" is more idiomatic since we're using MLIR as the primary implementation vehicle.

Reply:
Agreed. Done.